third_party/rust-hypervisor-firmware-subset/README.md

```diff
@@ -0,0 +1,8 @@
+This crate contains code extracted from the Rust Hypervisor Firmware code base:
```

Comment:
I am unsure what the correct procedure is when using only a subset from project, but normally the first commit should be a pristine copy from the source project which is then modified in future PRs. @tiziano88 any thoughts?

Reply:
I moved the initial import into https://github.com/project-oak/oak/pull/2786.